
package es.caib.emiservbackoffice.ws.api;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.2.12.redhat-00001
 * 2021-11-18T18:58:38.911+01:00
 * Generated source version: 3.2.12.redhat-00001
 */

@WebFault(name = "WsI18NError", targetNamespace = "http://impl.ws.emiservbackoffice.caib.es/")
public class WsI18NException extends Exception {

    private es.caib.emiservbackoffice.ws.api.WsI18NError wsI18NError;

    public WsI18NException() {
        super();
    }

    public WsI18NException(String message) {
        super(message);
    }

    public WsI18NException(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public WsI18NException(String message, es.caib.emiservbackoffice.ws.api.WsI18NError wsI18NError) {
        super(message);
        this.wsI18NError = wsI18NError;
    }

    public WsI18NException(String message, es.caib.emiservbackoffice.ws.api.WsI18NError wsI18NError, java.lang.Throwable cause) {
        super(message, cause);
        this.wsI18NError = wsI18NError;
    }

    public es.caib.emiservbackoffice.ws.api.WsI18NError getFaultInfo() {
        return this.wsI18NError;
    }
}
